packages/@okta/idx-js/idx.js

```diff
@@ -0,0 +1,2 @@
xt){case 0:return o.abrupt("return",f()(n,{method:t.method,headers:{"content-type":t.accepts},body:u()(X({},e,{},r))}).then((function(t){return t.ok?t.json():a.a.reject(t)})).then((function(t){return at(t)})));case 1:case"end":return o.stop()}}))}},rt=function(t){var r=t.refresh?tt:Z,n=function(t){t=V()(t)?t:[t];var r={},n={},e=!0,o=!1,i=void 0;try{for(var a,c=W()(t);!(e=(a=c.next()).done);e=!0){var u=a.value,s=K(u),f=s.neededParamsForAction,p=s.existingParamsForAction;r[u.name]=f,n[u.name]=p}}catch(t){o=!0,i=t}finally{try{e||null==c.return||c.return()}finally{if(o)throw i}}return{neededParams:r,existingParams:n}}(t),e=n.neededParams,o=r(t,n.existingParams[t.name]);return o.neededParams=e[t.name],o};function nt(t,r){var n=R()(t);if(A.a){var e=A()(t);r&&(e=C()(e).call(e,(function(r){return E()(t,r).enumerable}))),n.push.apply(n,e)}return n}var et=q()(G()(Q=["remediation","context"]).call(Q,(function(t){return[t,!0]}))),ot=function(t){var r,n,e={},o={};return m()(r=C()(n=R()(t)).call(n,(function(t){return!et[t]}))).call(r,(function(r){var n;if("object"===M()(t[r])&&!!t[r])if(t[r].rel)e[t[r].name]=rt(t[r]);else{var i=t[r],a=i.value,c=i.type,u=F()(i,["value","type"]);o[r]=function(t){for(var r=1;r<arguments.length;r++){var n,e=null!=arguments[r]?arguments[r]:{};if(r%2)m()(n=nt(e,!0)).call(n,(function(r){D()(t,r,e[r])}));else if(j.a)w()(t,j()(e));else{var o;m()(o=nt(e)).call(o,(function(r){b()(t,r,E()(e,r))}))}}return t}({type:c},u),"object"===c?(o[r].value={},m()(n=y()(a)).call(n,(function(t){var n,i=h()(t,2),a=i[0],c=i[1];c.rel?e[_()(n="".concat(r,"-")).call(n,a.name||a)]=rt(c):o[r].value[a]=c}))):o[r].value=a}else o[r]=t[r]})),{context:o,actions:e}},it=function(t){var r,n,e=(n=(null===(r=t.remediation)||void 0===r?void 0:r.value)||[],q()(G()(n).call(n,(function(t){return[t.name,rt(t)]})))),o=ot(t);return{remediations:e,context:o.context,actions:o.actions}},at=function(t){var r,n=it(t),e=n.remediations,i=n.context,c=n.actions,u={};m()(r=y()(e)).call(r,(function(t){var r=h()(t,2),n=r[0],e=r[1];u[n]=e.neededParams}));return{proceed:function(t){var r,n=arguments;return o.a.async((function(o){for(;;)switch(o.prev=o.next){case 0:if(r=n.length>1&&void 0!==n[1]?n[1]:{},e[t]){o.next=3;break}return o.abrupt("return",a.a.reject("Unknown remediation choice: [".concat(t,"]")));case 3:return o.abrupt("return",e[t](r));case 4:case"end":return o.stop()}}))},neededToProceed:u,actions:c,context:i,rawIdxState:t}};r.default={start:function(t){var r,n,e,i;return o.a.async((function(c){for(;;)switch(c.prev=c.next){case 0:if(r=t.domain,n=t.stateHandle){c.next=3;break}return c.abrupt("return",a.a.reject({error:"stateHandle is required"}));case 3:if(r){c.next=5;break}return c.abrupt("return",a.a.reject({error:"domain is required"}));case 5:return c.next=7,o.a.awrap(p({domain:r,stateHandle:n}).catch((function(t){return a.a.reject({error:"introspect call failed",details:t})})));case 7:return e=c.sent,i=at(e),c.abrupt("return",i);case 10:case"end":return c.stop()}}))}}}])}));
```

Comment:
may be add the commit id of  version we are consuming.

Reply:
yes, I have a [PR](https://github.com/okta/okta-idx-js/pull/8) for a small idx-js fix. once that is merged I'll include that commit as a reference.